protoc-gen-hack/plugin.go

```diff
@@ -172,6 +174,23 @@ func toPhpName(ns, name string) (string, string) {
 	return strings.Replace(ns, ".", "\\", -1), strings.Replace(name, ".", "_", -1)
 }
 
+func isReservedClassName(name string) bool {
+	lowerName := strings.ToLower(name)
+	for _, keyword := range reservedKeywords {
+		if lowerName == keyword {
+			return true
+		}
+	}
+	return false
+}
+
+func escapeReservedClassName(name string) string {
+	if isReservedClassName(name) {
+		return specialPrefix + name
```

Comment:
Lets use `pb_` instead of `XXX_` here. The protobuf convention I have observed in other implementations is XXX_ is for unstable features or parts of the API you are forced to expose but don't want people to depend on. pb_ is often used in the case (where you rewrite a name because it has collided with a language keyword).

I'm know I am not following that elsewhere but probably should be ;)

Reply:
Done.